fe/fe-core/src/main/java/com/starrocks/external/hive/MockHiveRepository.java

```diff
@@ -0,0 +1,338 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.external.hive;
+
+import com.clearspring.analytics.util.Lists;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import com.starrocks.catalog.HiveMetaStoreTableInfo;
+import com.starrocks.catalog.PartitionKey;
+import com.starrocks.common.DdlException;
+import com.starrocks.common.util.DateUtils;
+import org.apache.commons.collections4.map.CaseInsensitiveMap;
+import org.apache.hadoop.hive.metastore.api.FieldSchema;
+import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
+import org.apache.hadoop.hive.metastore.api.Table;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import static com.starrocks.sql.optimizer.Utils.getLongFromDateTime;
+import static java.lang.Double.NEGATIVE_INFINITY;
+import static java.lang.Double.POSITIVE_INFINITY;
+
+public class MockHiveRepository extends HiveRepository {
```

Comment:
You are my angel. but bad news bro, I will remove  the `HiveRepository` and `HiveMetaStoreTableInfo` in next version :(

Reply:
😭